integration-tests/test-cases/scenario-tests/src/test/java/io/cellery/integration/scenario/tests/petstore/PetStoreTestCase.java

```diff
@@ -0,0 +1,230 @@
+                "as expected");
+    }
+
+    /**
+     * This tests sign in for user Alice.
+     * @throws InterruptedException
+     */
+    @Test
+    public void signInAlice() throws InterruptedException {
+        this.signIn(this.alice);
+    }
+
+    /**
+     * This tests add to cart for user Alice.
+     * @throws InterruptedException
+     */
+    @Test
+    public void addToCartAlice() throws InterruptedException {
+        for (int i = 0; i < itemsOfAlice.length; i++) {
+            TimeUnit.SECONDS.sleep(10);
+            this.cart.addToCart(itemsOfAlice[i]);
+        }
+        validateWebPage("2", this.cart.getNumberOfItems(), "Pet store cart content before checkout is " +
+                "not as expected");
+    }
+
+    /**
+     * This tests checkout cart for user Alice.
+     * @throws InterruptedException
+     */
+    @Test
+    public void checkoutCartAlice() throws InterruptedException {
+        this.cart.checkout();
+        TimeUnit.SECONDS.sleep(10);
+        validateWebPage("0", this.cart.getNumberOfItems(), "Pet store cart content after checkout is " +
+                "not as expected");
+    }
+
+    /**
+     * This tests check order in for user Alice.
+     * @throws InterruptedException
+     */
+    @Test
+    public void checkOrdersAlice() throws InterruptedException {
+        TimeUnit.SECONDS.sleep(10);
+        validateWebPage("$ 675.00", this.order.getOrderValue(), "Pet store cart content after checkout is " +
+                "not as expected");
+    }
+
+    /**
+     * This tests sign out for user Alice.
+     */
+    @Test
+    public void signOutAlice() {
+        this.signOut(this.alice);
+    }
+
+    /**
+     * This tests sign in for user Bob.
+     * @throws InterruptedException
+     */
+    @Test
+    public void signInBob() throws InterruptedException {
+        this.signIn(this.bob);
+    }
+
+    /**
+     * This tests check order in for user Bob.
+     * @throws InterruptedException
+     */
+    @Test
+    public void checkOrdersBob() throws InterruptedException {
+        TimeUnit.SECONDS.sleep(10);
+        validateWebPage(Constants.PET_STORE_NO_ORDERS_PLACED, this.order.getOrderValue(),
+                "Pet store cart content after checkout is not as expected");
+    }
+
+    /**
+     * This tests sign out for user Bob.
+     */
+    @Test
+    public void signOutBob() {
+        this.signOut(this.bob);
+    }
+
+    @Test
+    public void terminate() throws Exception {
+        terminateCell(backEndInstanceName);
+        terminateCell(frontEndInstanceName);
+    }
+
+    @AfterClass
+    public void cleanup() {
+        webDriver.close();
+        try {
+            terminateCell(backEndInstanceName);
+            terminateCell(frontEndInstanceName);
+        } catch (Exception ignored) {
+        }
+    }
+
+    public void signIn(User user) throws InterruptedException {
```

Comment:
why this method is public? Looks like this is only used in this class.

Reply:
Changed the access modifier to private.